package.json

```diff
@@ -45,8 +45,7 @@
     "react-scripts": "1.0.17"
   },
   "scripts": {
-    "start":
-      "concurrently --kill-others \"nodemon --watch server server/index.js\" \"react-scripts start\"",
+    "start": "react-scripts start",
```

Comment:
This is no longer necessary, change back to: "start":
 -      "concurrently --kill-others \"nodemon --watch server server/index.js\" \"react-scripts start\""